framework/include/utils/InputParameters.h

```diff
@@ -750,19 +750,22 @@ class InputParameters : public Parameters
    */
   std::set<std::string> reservedValues(const std::string & name) const;
 
+  ///@{
   /**
    * Get/set a string representing the location (i.e. filename,linenum) in the input text for the
    * block containing parameters for this object.
    */
   std::string & blockLocation() { return _block_location; };
+  const std::string & blockLocation() const { return _block_location; };
+  ///@}
 
   ///@{
   /**
    * Get/set a string representing the full HIT parameter path from the input file (e.g.
    * "Mesh/foo") for the block containing parameters for this object.
    */
-  std::string & blockFullpath() { return _block_fullpath; }
-  const std::string & blockFullpath() const { return _block_fullpath; }
+  std::string & blockFullpath() { return _block_fullpath; };
+  const std::string & blockFullpath() const { return _block_fullpath; };
```

Comment:
remove these semicolons too.

Reply:
I f you want I can remove them from the whole file... there are a couple of other places in which the semicolon is after curly brace terminated blocks!